docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md

```diff
@@ -127,7 +127,7 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
 |`Switch.UseLegacyAccessibilityFeatures.2`|Controls whether accessibility features available in .NET Framework 4.7.2 are enabled (`false`) or disabled (`true`). If `true`, `Switch.UseLegacyAccessibilityFeatures` must also be `true` to enable .NET Framework 4.7.1 accessibility features.|.NET Framework 4.7.2|
 |`Switch.UseLegacyAccessibilityFeatures.3`|Controls whether accessibility features introduced in .NET Framework 4.8 are enabled (`false`) or disabled (`true`). If `true`, `Switch.UseLegacyAccessibilityFeatures` and `Switch.UseLegacyAccessibilityFeatures.2` must also be `true`.|.NET Framework 4.8|
 |`Switch.UseLegacyToolTipDisplay`|Controls whether tooltips are displayed when a user hovers the mouse cursor over a WPF control (`true`), or whether they are displayed both on keyboard focus and via keyboard shortcut key (`false`, the default behavior). For applications running on .NET Framework 4.8 but targeting previous versions of the .NET Framework, enabling both keyboard focus and shortcut key support requires that `Switch.UseLegacyAccessibilityFeatures`, `Switch.UseLegacyAccessibilityFeatures.2`, and `Switch.UseLegacyAccessibilityFeatures.3` all be set to `false`.|.NET Framework 4.8|
-|`System.Xml.`<br /><br /> `IgnoreEmptyKeySequences`|Controls whether empty key sequences in compound keys are ignored by XSD schema validation. For more information, see [Mitigation: XML Schema Validation](../../../migration-guide/mitigation-xml-schema-validation.md).|.NET Framework 4.6|  
+|`Switch.System.Xml.`<br /><br /> `IgnoreEmptyKeySequences`|Controls whether empty key sequences in compound keys are ignored by XSD schema validation. For more information, see [Mitigation: XML Schema Validation](../../../migration-guide/mitigation-xml-schema-validation.md).|.NET Framework 4.6|  
```

Comment:
```suggestion
|`Switch.System.Xml`<br /><br /> `IgnoreEmptyKeySequences`|Controls whether empty key sequences in compound keys are ignored by XSD schema validation. For more information, see [Mitigation: XML Schema Validation](../../../migration-guide/mitigation-xml-schema-validation.md).|.NET Framework 4.6|  
```
?

Reply:
The period is needed

Reply:
But your comment made me realize that this was poorly formatted.
I've removed the extra line

![image](https://user-images.githubusercontent.com/12971179/71214911-38eb2100-226b-11ea-835c-9ba5e999d1e0.png)